packages/@o3r/workspace/schematics/index.it.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('new otter workspace', () => {
   test('should add sdk to an existing workspace', () => {
     const { workspacePath, isInWorkspace, untouchedProjectsPaths } = o3rEnvironment.testEnvironment;
     const execAppOptions = {...getDefaultExecSyncOptions(), cwd: workspacePath};
-    packageManagerExec({script: 'ng', args: ['g', 'sdk', '@my-sdk/sdk']}, execAppOptions);
+    packageManagerExec({script: 'ng', args: ['g', 'sdk', 'my-sdk']}, execAppOptions);
```

Comment:
this change is not mandatory, right ?

Reply:
This change is not mandatory indeed but here to avoid create a new integration test to check when the user provide a sdk name not following the asked format.